app/src/main/java/sdumchykov/androidApp/presentation/viewPager/contacts/fetchContacts/FetchContacts.kt

```diff
@@ -1,34 +1,36 @@
 package sdumchykov.androidApp.presentation.viewPager.contacts.fetchContacts
 
 import android.Manifest
-import android.annotation.SuppressLint
 import android.app.Activity
-import android.provider.ContactsContract
 import android.widget.Toast
 import com.karumi.dexter.Dexter
 import com.karumi.dexter.PermissionToken
 import com.karumi.dexter.listener.PermissionDeniedResponse
 import com.karumi.dexter.listener.PermissionGrantedResponse
 import com.karumi.dexter.listener.PermissionRequest
 import com.karumi.dexter.listener.single.PermissionListener
-import sdumchykov.androidApp.domain.model.UserModel
-import sdumchykov.androidApp.domain.utils.Constants
-import sdumchykov.androidApp.presentation.viewPager.contacts.MyContactsViewModel
 
-class FetchContacts(private val parentViewModel: MyContactsViewModel) {
+class FetchContacts {
 
-    fun fetchContacts(activity: Activity) {
+    fun fetchContacts(
+        activity: Activity,
```

Comment:
you don't need an activity to show toasts. You only need contexts

Reply:
But i should pass value of type Activity to Dexter.withActivity(activity:Activity!).